### DISCUSSION TITLE
Is packbits compression supported?

### DISCUSSION THREAD

**Discussion by jessjaco** [tlnagy/TiffImages.jl#90] [Q&A]
Does TiffImages support packbits compression? It's part of the 6.0 baseline spec but I can't seem to open a packbits compressed file.

**Comment by tlnagy** [comment]
Sorry for the delayed response, but basic packbits compressed images are supported: https://github.com/tlnagy/TiffImages.jl/blob/562225eb79da2b0e5667b30aa0785b61ea0892ea/src/compression.jl#L12-L29

Did you get an error message when you loaded the image? Could you share that? If you want, you can send me a link to an example TIFF file to the email listed on my profile page and I could take a look.
